.github/workflows/validate_datasets.yml

```diff
@@ -73,7 +73,9 @@ jobs:
         for i in $(ls -d */ | grep -v node_modules);
         do
           echo "Validating dataset" $i
-          if [ -f ${i%%/}/.bids-validator-config.json ]; then
+          if [ -f ${i%%/}/.SKIP_VALIDATION ]; then
+            echo "skipping validation for ${i%%/}"
```

Comment:
```suggestion
            echo "skipping validation for ${i%%/}"; rc=0
```

Reply:
It seems to work as expected without `rc=0` --> the following datasets continue to be evaluated, but ds000001-fmriprep is properly skipped and for that ds, only an echoed str is printed.